source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -1073,13 +1079,17 @@ TEST_F(NiDAQmxDriverApiTests, AOVoltageChannel_WriteAODataWithOutOfRangeValue_Re
   create_ao_voltage_chan(AO_MIN, AO_MAX);
 
   start_task();
-  auto write_data = generate_random_data(AO_MIN, AO_MAX, 100);
-  write_data[80] += AO_MAX;
-  WriteAnalogF64Response write_response;
-  auto status = write_analog_f64(write_data, write_response);
+  try {
+    auto write_data = generate_random_data(AO_MIN, AO_MAX, 100);
+    write_data[80] += AO_MAX;
+    WriteAnalogF64Response write_response;
+    write_analog_f64(write_data, write_response);
+    EXPECT_FALSE(true);
```

Comment:
There's a more direct way to do this:
https://google.github.io/googletest/reference/assertions.html#success-failure

```
FAIL() << "We shouldn't get here.";
```